client/src/api/robot.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @category Client
+ * @module
+ */
+
+/**
+ * @description Fetch tasklist from Mongo-DB
+ * @param {String} application - String with currently selected application from ApplicationDropdown
+ */
+const setRobotJob = async (robotId) => {
+  const response = await fetch(`/robot/jobs/add?robotId=${robotId}`);
```

Comment:
maybe the route should not be /robots/jobs/... but rather just /jobs/...

Reply:
✅